catalog/core/catalog-core-commons/src/main/java/ddf/catalog/filter/delegate/TagsFilterDelegate.java

```diff
@@ -57,7 +57,7 @@ public Boolean and(List<Boolean> operands) {
 
   @Override
   public Boolean or(List<Boolean> operands) {
-    return operands.stream().allMatch(op -> op);
+    return operands.stream().anyMatch(op -> op);
```

Comment:
I don't think this is correct. The original intent of this filter was to test to make sure all filter paths included a tag component. This change causes there to be a hole in the logic where the query `title like 'test' OR 'metacard-tags like 'something'` would return true. Unless the catalog framework has been updated (entirely possible) this query would return versioned metacards as well as standard metacards with 'test' in the title.